src/storage/v2/replication/replication_persistence_helper.hpp

```diff
@@ -23,18 +23,23 @@
 
 namespace memgraph::storage::replication {
 
-struct ReplicaStatus {
+static const std::string kReservedReplicationRoleName{"__replication_role"};
```

Comment:
Is there some reason not to use inline constexpr?

Reply:
changed to string_view and inline